website/events/templates/events/event.html

```diff
@@ -16,7 +16,20 @@
 {% block body %}
     <section class="page-section" id="events-detail">
         <div class="container">
-            <h1 class="section-title">{{ event.title }}</h1>
+
+            <h1 class="section-title">
+                {{ event.title }}
+                {% if permissions.manage_event %}
+                    <a href="{% url 'admin:events_event_change' event.pk %}"
```

Comment:
Actually, could you change it to go to e.g. "http://127.0.0.1:8000/admin/events/event/11/details/"?
```suggestion
                    <a href="{% url 'admin:events_event_details' event.pk %}"
```
I think this should do it